concent-builder/configure-jenkins.yml

```diff
@@ -0,0 +1,184 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Clone concent repositories
+          git:
+            repo:    "{{ item.url }}"
+            dest:    "{{ jenkins_home_dir }}/{{ item.name }}"
+            clone:   yes
+            update:  yes
+            version: dev
+            force:   yes
+          with_items:
+          #  - { url: "{{ repositories['concent-deployment'].url }}", name: concent-deployment }
+            - { url: "https://github.com/bartoszbetka/concent-deployment.git", name: concent-deployment }
```

Comment:
The URL should not be hard-coded.

Reply:
As above. The commented version is correct, the forked repository is for testing purpose.

Reply:
We have the hook. This should be changed now.